tbprobe.h

```diff
@@ -53,6 +53,10 @@ extern "C" {
 #    endif
 #  endif
 
+#  ifndef __WIN32__
+#    define MemoryBarrier __asm__ __volatile__("":::"memory");
+#  endif
```

Comment:
Not sure if `MemoryBarrier` macro on windows i the same as `__asm__ __volatile__("":::"memory")`.
I didn't lookup the semantics.